registry/src/main/java/io/terrakube/registry/service/module/ModuleServiceImpl.java

```diff
@@ -138,33 +138,11 @@ public void updateModuleDownloadCount(String organizationName, String moduleName
         terrakubeClient.updateModule(moduleRequest, organizationId, module.getId());
     }
 
-    private String getAccessToken(String organizationId, String vcsId, String repository_source) {
-        Vcs vcs = getVcsInformation(organizationId, vcsId);
-        if (vcs == null)
-            return null;
-        String token = vcs.getAttributes().getAccessToken();
-        if (token == null && vcs.getAttributes().getConnectionType().equals("STANDALONE")) {
-            log.info("The VCS connection is on a standalone app, getting the GitHub App token");
-            GitHubAppToken gitHubAppToken = getGitHubAppTokenInformation(vcs.getAttributes().getClientId(), repository_source);
-            token = gitHubAppToken.getAttributes().getToken();
-        }
-        return token;
-    }
-
     private Vcs getVcsInformation(String organizationId, String vcsId) {
         return terrakubeClient.getVcsById(organizationId, vcsId).getData();
     }
 
     private Ssh getSshInformation(String organizationId, String sshId) {
         return terrakubeClient.getSshById(organizationId, sshId).getData();
     }
-    
-    private GitHubAppToken getGitHubAppTokenInformation(String vcsClientId, String repository_source) {
-        URI uri = URI.create(repository_source);
-        String owner = uri.getPath().split("/")[1];
-        List<GitHubAppToken> gitHubAppTokens = terrakubeClient.getGitHubAppTokenByVcsIdAndOwner(owner, vcsClientId).getData();
-        if (gitHubAppTokens.isEmpty())  return null;
-
-        return gitHubAppTokens.get(0);
-    }
```

Comment:
As far as I can understand, this code is just working around the fact that the ordinary token refresh is not implemented for pki-based apps.